views/show.erb

```diff
@@ -0,0 +1,17 @@
+<h2><a href="/pokemons">View all Pokemon</a></h2>
+
+<h3>
+  name: <%= @pokemon.name %>
+</h3>
+
+<p>
+  cp: <%= @pokemon.cp %>
+</p>
+
+<p>
+  Poke Type: <%= @pokemon.poke_type %>
+</p>
+
+<img src="<%= @pokemon.img_url %>" alt="<%= @pokemon.name %>">
+
+<a href="/pokemons/<%= @pokemon.id %>/edit">Edit Pokemon Info</a>
```

Comment:
Everything looks good, Pam. Glad to see you are comfortable with this lab. I don't have much to comment about since you've set everything up correctly and logically.